src/Agent.Sdk/SecretMasking/LoggedSecretMasker.cs

```diff
@@ -4,38 +4,28 @@
 using System;
 using ValueEncoder = Microsoft.TeamFoundation.DistributedTask.Logging.ValueEncoder;
 using ISecretMaskerVSO = Microsoft.TeamFoundation.DistributedTask.Logging.ISecretMasker;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
 
 namespace Agent.Sdk.SecretMasking
 {
     /// <summary>
     /// Extended secret masker service, that allows to log origins of secrets
     /// </summary>
-    public class LoggedSecretMasker : ILoggedSecretMasker
+    public class LoggedSecretMasker : SecretMasker, ILoggedSecretMasker
```

Comment:
>SecretMasker [](http://example.com/codeflow?start=38&length=12)

The logged secret masker (as opposed to the legacy logged secret masker) can extend `SecretMasker` rather than wrapping it (because no other `ILoggedSecretMasker` will be wrapped by this class). This change will help us switch over to using a `SecretMasker` instance that 1ES provides that lives in a different repo ([https://github.com/microsoft/security-utilities](https://github.com/microsoft/security-utilities)).